### PR TITLE
CLOUD-513 Add license tests

### PR DESCRIPTION
**Pull request by hors**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| self-healing | passed |
| storage | passed |
| init-deploy | passed |
| demand-backup | failed |
| monitoring | passed |
| scheduled-backup | failed |
| limits | passed |
| upgrade | failed |
| scaling | passed |
| operator-self-healing | passed |
| monitoring-2-0 | passed |
| arbiter | passed |
| one-pod | failed |
| service-per-pod | passed |
| security-context | failed |
| upgrade-consistency | failed |

File: Jenkinsfile [hunk 1]
```diff
@@ -173,7 +180,44 @@ pipeline {
...
+                expression {
+                    !skipBranchBulds
+                }
+            }
+            steps {
+                sh '''
+                    mkdir -p $WORKSPACE/src/github.com/percona
+                    ln -s $WORKSPACE $WORKSPACE/src/github.com/percona/percona-server-mongodb-operator
+                    sg docker -c "
+                        docker run --rm -v $WORKSPACE/src/github.com/percona/percona-server-mongodb-operator:/go/src/github.com/percona/percona-server-mongodb-operator -w /go/src/github.com/percona/percona-server-mongodb-operator -e GO111MODULE=on golang:1.13 sh -c 'go mod init && go build ./... && rm -rf Gopkg.lock Gopkg.toml vendor && go build -v ./... && go build -o percona-server-mongodb-operator github.com/percona/percona-server-mongodb-operator/cmd/manager'
```
Comment (delgod) [inline Jenkinsfile]:
```suggestion
                        docker run \
                            --rm \
                            -v $WORKSPACE/src/github.com/percona/percona-server-mongodb-operator:/go/src/github.com/percona/percona-server-mongodb-operator \
                            -w /go/src/github.com/percona/percona-server-mongodb-operator \
                            -e GO111MODULE=on \
                            golang:1.13 sh -c ' \
                                go mod init \
                                    && go build ./... \
                                    && rm -rf Gopkg.lock Gopkg.toml vendor \
                                    && go build -v ./... \
                                    && go build -o percona-server-mongodb-operator github.com/percona/percona-server-mongodb-operator/cmd/manager 
                            '
```

File: Jenkinsfile [hunk 2]
```diff
@@ -138,6 +138,13 @@ pipeline {
                         | sudo tar -C /usr/local/bin --strip-components 1 -zvxpf -
                     curl -s -L https://github.com/openshift/origin/releases/download/v3.11.0/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz \
                         | sudo tar -C /usr/local/bin --strip-components 1 --wildcards -zxvpf - '*/oc'
+                    
+                    curl -s -L https://github.com/mitchellh/golicense/releases/latest/download/golicense_0.2.0_linux_x86_64.tar.gz \
+                        | sudo tar -C /usr/local/bin --wildcards -zxvpf -
+                    curl -s -L https://github.com/src-d/go-license-detector/releases/latest/download/license-detector.linux_amd64.gz \
+                        | gunzip > license-detector
```
Comment (delgod) [inline Jenkinsfile]:
```suggestion
                        | gunzip > /usr/local/bin/license-detector
```

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

File: Jenkinsfile (hunk 1 above)
Comment (hors) [reply Jenkinsfile]: done

File: Jenkinsfile (hunk 2 above)
Comment (hors) [reply Jenkinsfile]: done

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| storage | failed |
| self-healing | passed |
| demand-backup | failed |
| monitoring | failed |
| scheduled-backup | failed |
| limits | passed |
| monitoring-2-0 | passed |
| scaling | passed |
| upgrade | failed |
| operator-self-healing | passed |
| arbiter | passed |
| one-pod | failed |
| service-per-pod | passed |
| security-context | failed |
| upgrade-consistency | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| self-healing | passed |
| storage | passed |
| init-deploy | passed |
| demand-backup | failed |
| monitoring | passed |
| scheduled-backup | failed |
| limits | passed |
| scaling | passed |
| monitoring-2-0 | passed |
| upgrade | failed |
| operator-self-healing | passed |
| arbiter | passed |
| one-pod | failed |
| service-per-pod | passed |
| security-context | failed |
| upgrade-consistency | failed |

**Pull request merged**